php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -177,7 +177,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      * instrument.
      *
      * @param string  $instrument       The name of the instrument to use
-     * @param string  $commentID        The CommentID identifying the data to load
+     * @param ?string $commentID        The CommentID identifying the data to load
```

Comment:
This does not have to be changed.

Reply:
The `setup` functions in `NDB_BVL_Instruments` and `NDB_BVL_Instruments_LINST` accepts null values for `$commentID` and `$page`. I'm removing the changes not to break anything though, as you mentioned in another comment, but ideally shouldn't it be consistent one way or the other?